articles/vmware/vmw-how-access-centos-update-servers.md

```diff
@@ -35,49 +35,41 @@ Our Elevated OFFICIAL security domain doesn't natively connect to the internet,
 
 ### Option 1. UKCloud-managed repository servers
 
-We provide repositories for CentOS 6 on our Elevated OFFICIAL security domain. To access them:
+We provide repositories for CentOS 6 and 7 on our Elevated OFFICIAL security domain. To access them:
 
 Create a file called `il3-repos.repo` in `/etc/yum.repos.d`, and populate it with the following:
```

Comment:
Reword to

For CentOS base files, created a file called il3-repos.repo in /etc/yum.repos.d and populate it with the following